Blaise-Shyaka/Politico
# [#171591992] A page to create new political party

# What does this PR do?

It makes a new page allowing the admin to create a new political party

# Description of tasks to be completed

- Create a page with a form
- The form should have 3 input fields; party name, headquarters, and motto.
- Make the page responsive at least on mobile, tablet, and desktop

# How should this be manually tested?

- Clone this repository and `cd /templates/html/`
- Open `create-political-party.html`
- Verify the presence of 3 fields; namely, party name, party motto, and party headquarters

# What are the relevant pivotal tracker stories?

#171591992

# Screenshots

![Screenshot (45)](https://user-images.githubusercontent.com/52489421/75981663-a06d5000-5eed-11ea-8fe7-b72cb4871771.png)
![Screenshot (46)](https://user-images.githubusercontent.com/52489421/75981670-a6633100-5eed-11ea-9140-ad85c637109b.png)
![Screenshot (47)](https://user-images.githubusercontent.com/52489421/75981681-ab27e500-5eed-11ea-9c3d-6a8b17c81e5b.png)
![Screenshot (48)](https://user-images.githubusercontent.com/52489421/75981690-b0852f80-5eed-11ea-8b1f-b40f5c237fb5.png)
![Screenshot (49)](https://user-images.githubusercontent.com/52489421/75981700-b5e27a00-5eed-11ea-9936-152fed40a249.png)
